structuredheadings/plugin.js

```diff
@@ -594,6 +658,56 @@
             setStyle(editor, node, style);
           }
         }
+      },
+
+      applyPresetToList: {
+        getPresetStyleArray: function (editor, presetName) {
+          if (presetName === "clear") {
+            return [editor.config.listClassMappings.clear];
+          }
+
+          var styleArray = presetName.split(".").map(function (str) {
+            var key = str.trim()[0];
+            return editor.config.listClassMappings[key];
+          }).filter(function (str) {
+            if (str) {
+              // also purge empty strings, undef, etc. because I am lazy
+              return true;
```

Comment:
filter works off of true/false (although tbh I don't recall if returning a falsy undef is sufficient)